### ISSUE TITLE
Mapping SSR

### ISSUE THREAD

**Issue by arzoo707** [lmdu/krait#20]
One of the Reviewers raised a query "distribution of identified SSRs in different genomic regions". I tried to run on both Linux and Windows operating systems. Identification of SSRs and their statistical report was perfectly obtained but while analyzing annotation files no results were obtained. I downloaded the GFF, GTF, and GFF3 files from NCBI and Ensemble. Please guide me regarding this issue.  Thanking you
![Screenshot (30)](https://user-images.githubusercontent.com/84401942/118687173-576b9800-b822-11eb-9e44-220b022e576d.png)
![Screenshot (31)](https://user-images.githubusercontent.com/84401942/118687181-59355b80-b822-11eb-86da-f52b4d78488b.png)

**Comment by lmdu** [comment]
You can check the sequence name is the same or not in your annotation file and sequence file.

**Comment by arzoo707** [comment]
Respected sir
In this case we cannot examine location of identified SSRs in other
genomes?
Like if i identify SSRs in Grevillea robusta we cannot search the
distribution of these identified SSRs in Eucalyptus annotation file?

**Comment by lmdu** [comment]
Yes, the mapping function in our program is just comparing the coordinates of SSRs with gene features not alignment to locate SSRs to genes.
